lib/mquery.js

```diff
@@ -1402,12 +1402,18 @@ function _pushArr (opts, field, value) {
  * limit, skip, maxScan, batchSize, comment
  *
  * Sets these associated options.
+ * With mongodb>=3.2.6 if the value is a string doesn't accept it, converting type string numbers to numeric type in applicable cases
  *
  *     query.comment('feed query');
  */
 
 ;['limit', 'skip', 'maxScan', 'batchSize', 'comment'].forEach(function (method) {
   Query.prototype[method] = function (v) {
+    if (method !== 'comment') {
+      let value = Number(v);
```

Comment:
General idea is that node 0.10 and 0.12 don't support anything from ECMAScript 2015 / ES6, node 4 supports some, node 5 supports a little more, etc. TBH it's mostly just trial and error for me, just run node v0.10 and try using Number.isNan() :)